content/en/docs/Reference/operator.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Operators"
+linkTitle: "Operators"
+weight: 3
+date: 2021-03-15
+---
+## What is an Operator?
+
+Operators make it easy to manage complex stateful applications on top of Kubernetes. By [Extendig the Kubernetes API with CustomResourceDefinitions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) it is possible to create what we call Operator Projects where you will be able to create and define your own API's to represent your solutions on the cluster and managed them by looking for the desired state. For a better understanding see [Operator Pattern][operator-pattern]
+
+## What is the difference between a Controller and an Operator
+
+See that following the [Operator's pattern][operator-pattern]; you will create [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) which provides a reconcile function responsible for synchronizing the resources until a desired state on the cluster. For a more practical explanation, see also [What’s in a controller?](https://book.kubebuilder.io/cronjob-tutorial/controller-overview.html#whats-in-a-controller)
+
+## What is Operator Lifecycle Manager?
+
+Operator Lifecycle Manager (OLM) helps users install, update, and manage the lifecycle of all Operators and their associated services running across their clusters. This project is a component of the [Operator Framework][operator-framework], an open-source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the introduction [blog post](https://www.openshift.com/blog/introducing-the-operator-framework?extIdCarryOver=true&sc_cid=701f2000001Css5AAC).
+
+## How does OLM enable cluster admins and Developers?
+
+The OLM is enabled by default in OpenShift Container Platform 4.X, which aids cluster administrators in installing, upgrading, and granting access to Operators running on their cluster. The OpenShift Container Platform web console provides management screens for cluster administrators to install Operators, as well as grant specific projects access to use the catalog of Operators available on the cluster.
```

Comment:
```suggestion
The OLM is enabled by default in OpenShift Container Platform 4.x, which aids cluster administrators in installing, upgrading, and granting access to Operators running on their cluster. The OpenShift Container Platform (OCP) web console provides management screens for cluster administrators to install Operators, as well as grant specific projects access to use the catalog of Operators available on the cluster.
```

Reply:
Hmm, we may want to just document the full name, like Operator Lifecycle Manager, or OpenShift Container Platform once, and then abbreviate the rest of those references throughout this document. WDYT?

Reply:
I have checked for both. there arent many repetitions and also some of them are in the heading

Reply:
Cool, just wanted to double-check. Thanks :+1:.